tests/foreman/ui/test_rhcloud_inventory.py

```diff
@@ -28,6 +28,8 @@
 from robottelo.utils.io import get_remote_report_checksum
 from robottelo.utils.io import get_report_data
 
+# import epdb
```

Comment:
Remove the debugging, the pre-commit must have identified this , but it did not cause its commented !! :)